src/main.py

```diff
@@ -1,14 +1,21 @@
-import parse_fasta
-import sam_to_fasta_file
+from pathlib import Path
+
+from src import parse_fasta
+from src import sam_to_fasta_file
 
 # this method starts the tasks for the whole exercise (1.4 and 2.2 to be precise)
 
 if __name__ == '__main__':
     print(parse_fasta.map_reads(
-        r"/Users/Aline/Desktop/21HS/21HS PrBi/PrBi Alexander Kanitz/PrBi/RNA-Seq/Test files/sequences.fasta",
-        r"/Users/Aline/Desktop/21HS/21HS PrBi/PrBi Alexander Kanitz/PrBi/RNA-Seq/Test files/genome.fasta"))
-    sam_to_fasta_file.sam_to_fasta_file('Aligned.out.sam')
-    dict = sam_to_fasta_file.parse_fasta.map_reads(
+        str(Path(__file__).parents[1] / "tests/test_files/sequences.fasta"),
```

Comment:
To avoid absolute paths, we are using paths here that are relative to the location of _this_ file, which you can get via the `__file__` variable (which Python sets automatically). From there we get the parent directory and so on... The `Path()` constructor from the `pathlib` module is quite useful for doing these file system operations, as it not only has convenient methods like `.parents`, but also has the `/` operator which allows constructing file paths in an OS-portable way (so, e.g., on Windows this would be interpreted as `\`). This also works if `/` is within a string as it is here in `tests/test_files/sequences.fasta`. Without `pathlib` this would fail on Windows!
```py
Path(__file__).parents[1] / "tests" / "test_files" / "sequences.fasta"
```